x-pack/packages/security/plugin_types_public/src/privileges/privileges_api_client.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import type { RawKibanaPrivileges } from '@kbn/security-authorization-core';
+import { RawKibanaPrivileges } from '@kbn/security-plugin-types-common';
```

Comment:
```suggestion
import type { RawKibanaPrivileges } from '@kbn/security-plugin-types-common';
```

Reply:
Ah, that reminded me that we don't apply our own eslint rules to all folders we own: fixing it in https://github.com/elastic/kibana/pull/195711

Reply:
I'm surprised this isn't a Kibana-wide rule. Do you recall why we need to opt into this?

Reply:
Not sure, if I read the config correctly, we only apply this rule in a handful of places today: https://github.com/search?q=repo%3Aelastic%2Fkibana%20%22%40typescript-eslint%2Fconsistent-type-imports%22&type=code. Maybe @elastic/kibana-operations is willing to promote `@typescript-eslint/consistent-type-imports` as a global rule?